sechub-doc/src/docs/asciidoc/sechub-quickstart-guide.adoc

```diff
@@ -52,13 +52,15 @@ NOTE: Tested with Alpine Linux 3.12, 3.13 and 3.14.
 
 WARNING: Cross-compiling the SecHub client on Alpine Linux 3.11 does not work (Go version go1.13.13 linux/amd64). 
 
-==== Debian
+==== Debian/Ubuntu
 
 ----
 sudo apt install openjdk-11-jdk-headless golang git curl jq
 ----
 
-NOTE: Tested with Debian 10 "Buster".
+NOTE: Tested with Debian 10 "Buster", Ubuntu 18.04 "Bionic" and 20.04 "Focal" LTS.
+
+WARNING: If the version of your Go Lang is 1.15<= please install a more recent version from https://go.dev/dl/[here].
```

Comment:
Golang 1.15+ is required otherwise it does not compile. Please make that clear in your description.

Reply:
Done!